infrastructure/templates/template.json

```diff
@@ -1276,6 +1276,7 @@
         "WEBSITE_RUN_FROM_PACKAGE": "1",
         "PublicStorage": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-storage-public'), '2018-02-14').secretUriWithVersion, ')')]",
         "enableSwagger": "[parameters('enableSwagger')]",
+        "PublicAppUrl": "[variables('publicAppUrl')]",
```

Comment:
I don't think that's necessary because the variable is already defined for use elsewhere in the app template, so must already be added to all environments.